app/Resources/public/css/print.css

```diff
@@ -119,3 +93,103 @@ table.table td {
     border: 1px solid #555;
     border-collapse: collapse;
 }
+
+/* print table pdf */
+.data_table{
+    width: 100%;
+    padding: 0;
+    border: 1px solid #b5cae7;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5266914)